worlds/witness/docs/setup_fr.md

```diff
@@ -0,0 +1,47 @@
+# Guide d'installation du Witness randomizer
+
+## Logiciels Requis
+
+- [The Witness pour Windows 64-bit (par exemple, la version Steam)](https://store.steampowered.com/app/210970/The_Witness/)
+- [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+
+## Logiciels Facultatifs
+
+- [ArchipelagoTextClient](https://github.com/ArchipelagoMW/Archipelago/releases)
+- [The Witness Map- et Auto-Tracker](https://github.com/NewSoupVi/witness_archipelago_tracker/releases), pour usage avec [PopTracker](https://github.com/black-sliver/PopTracker/releases)
+
+## Rejoindre un jeu multimonde
+
+1. Lancez The Witness
+2. Commencez une nouvelle partie
+3. Lancez [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+4. Inscrivez l'adresse Archipelago, votre nom de joueur et le mot de passe du jeu multimonde
+5. Cliquez sur "Connect"
+6. Jouez!
+
+Pour continuer un jeu multimonde précedemment commencé:
+
+1. Lancez The Witness
+2. Chargez la sauvegarde sur laquelle vous avez dernièrement joué ce monde, si ce n'est pas celle qui a été chargée automatiquement 
+3. Lancez [The Witness Archipelago Randomizer](https://github.com/NewSoupVi/The-Witness-Randomizer-for-Archipelago/releases/latest)
+4. Cliquez sur "Load Credentials" (ou tapez les manuellement)
+5. Cliquez sur "Connect"
+
+## Archipelago Text Client
+
+Il est recommandé d'utiliser le "Archipelago Text Client" en parallèle afin de suivre quels items vous envoyez et recevez.
+<br/>The Witness affiche également ces informations en jeu, mais seulement pour une courte période et donc il est facile de manquer ces messages.
+
+<br/><br/>Bien sûr, vous pouvez également utiliser l'auto-tracker!
```

Comment:
Google's markdown style discourage "'<br/>". Changing line does the job, so should be removed. (Do the same with English and Deustch)